src/components/Tags/Tags.vue

```diff
@@ -1,65 +1,88 @@
 <template>
   <div class="tags">
-      <Tag v-for="tag in internalTags"
-           :key="tag"
-           v-bind="{icon, disabled, outlined, hovered, elevated, flat, color}"
-      >{{ tag }}</Tag>
+    <i v-if="icon" class="pi tags__icon" :class="icon" />
+    <Tag flat>
+      {{ internalTags }}
+      <template v-if="tagAmountLimit">
+        <Tooltip  placement="bottom">
+          <template v-slot:trigger>
+            ...<span class="tags__moreTooltip">{{hiddenTagsNumber}} more</span>
+          </template>
+          <span class="tags__tooltipTag">{{ hiddenTags }}</span>
+        </Tooltip>
+      </template>
+    </Tag>
   </div>
 </template>
 
 <script lang="ts">
 import { defineComponent } from 'vue'
 import Tag from "@/components/Tag/Tag.vue";
+import Tooltip from "@/components/Tooltip/Tooltip.vue"
 
 export default defineComponent({
   name: "Tags",
-  components: { Tag },
+  components: { Tag, Tooltip },
   props: {
     icon: {
       type: String,
-      default: null
+      default: "pi-label"
     },
     tags: {
       type: Array,
       default: function () { return [] }
     },
-    disabled: {
-      type: Boolean,
-      default: false
-    },
-    outlined: {
-      type: Boolean,
-      default: false
+    limit: {
+      type: Number,
+      default: 4
+    }
+  },
+  computed: {
+    tagAmountLimit(): boolean {
+      return this.tags.length > this.limit
     },
-    elevated: {
-      type: Boolean,
-      default: false
+    visibleTags(): string {
+      return this.tags.slice(0, this.limit).join(", ")
     },
-    flat: {
-      type: Boolean,
-      default: false
+    hiddenTags(): string {
+      return this.tags.slice(this.limit).join(", ")
     },
-    hovered: {
-      type: Boolean,
-      default: false
+    hiddenTagsNumber(): number {
+      return this.tags.slice(this.limit).length
```

Comment:
I think we need to split out `this.tags.slice(this.limit)` into its own computed property that we then reference in these. That way we don't duplicate slicing the array.